### DISCUSSION TITLE
My 3 Zeon GT-1X Miners purchased on 1st June not yet Minted. Minted.

### DISCUSSION THREAD

**Discussion by jacobbronya** [blockscout/blockscout#10412] [Show and tell]
Please, leaders I need help to Mint and Mine my 3 Miners purchased on the 1st of June, 2024. Any help?

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
